src/cli/arch/cli.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+FILE=/usr/share/pam-configs/Facerec
+SUDO=/etc/pam.d/sudo
+TEXT='auth      sufficient   pam_python.so /lib/Auth/Facerec/pam.py'
```

Comment:
This won't work I think. The file will be `/usr/lib/Auth/Facerec/pam_ptn.py`

Reply:
/usr/lib is symlink to /lib

Reply:
but the file name is `pam_ptn.py`

Reply:
renamed